pom.xml

```diff
@@ -5,14 +5,13 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>com.skyevel</groupId>
-    <artifactId>UniversalScoreboard</artifactId>
+    <artifactId>universal-scoreboard</artifactId>
```

Comment:
Please revert it to the original artifact id to maintain backward compatibility.

Reply:
I get it, backwards compatibility... but the repo right only avaliable on Local M2 (This means you need to build it before you can use it as a dependency in Maven/Gradle) storage and the official Maven naming convention states that artifact id's should be lowercase without strange characters.

> - **artifactId** is the name of the jar without version. If you created it, then you can choose whatever name you want with lowercase letters and no strange symbols. If it's a third party jar, you have to take the name of the jar as it's distributed.
eg. `maven, commons-math`
> Quote taken from: https://maven.apache.org/guides/mini/guide-naming-conventions.html